portfolio/src/main/webapp/script.js

```diff
@@ -88,3 +88,53 @@ function createListElement(text) {
   return liElement;
 }
 
+function loadContacts(){
+  fetch('/list-contacts')
+  .then(response => response.json())
+  .then((contacts) => {
+    const taskListElement = document.getElementById('contact-list');
+    console.log(contacts);
+    contacts.forEach((contact) => {
+      taskListElement.appendChild(createContactElement(contact));
+     
+    })
+  });
+}
+function createContactElement(contact) {
+  const contactElement = document.createElement('li');
```

Comment:
It's a good practice to keep literals as constants at the top of the file.

Reply:
that does make sense but in the case of these variables are only needed within the function, I feel I should leave them here. Also, all the code examples did this as well.

Reply:
Sure. But these values ('li', 'Name: ", "Email: " etc) are not variables, they're constant literals.

Reply:
Makes sense will try to fix it lol

Reply:
Was this updated?

Also, github won't allow me to leave a comment there, but line 81 uses the literal 5. Can you store this in a constant at the top of this file. possibly with a comment describing what it's for?

Reply:
It should be updated now, with literals in the code now